stdlib/nodejs/pathname.rb

```diff
@@ -0,0 +1,31 @@
+class Pathname
+  include Comparable
+
+  @__path__ = `require('path')`
+  `var __path__ = #{@__path__}`
+
+  def initialize(path)
```

Comment:
@elia Am I forced to duplicate this code ? Is there a better way to do it ?

Reply:
I think would be savvy to `require 'pathname'` at the top of the file, and only overwrite where needed, if an extractable pattern arises then we can evaluate a pluggable system for different platforms (like it's done with `Opal .exit()`), but for now it's probably unnecessary.